src/main/java/com/ericsson/eiffel/remrem/publish/service/MessageServiceRMQImpl.java

```diff
@@ -168,4 +210,22 @@ private void createFailureResult(List<PublishResultItem> events) {
                 PropertiesConfig.INVALID_EVENT_CONTENT);
         events.add(event);
     }
+    
+    private void createUnsuccessfulEvents(JsonElement obj) {
```

Comment:
createUnsuccessfulEvents => addUnsuccessfulResultItem since it does not create, does not handle events and only handles one item.